src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
 
 export function ExplanationCard(props: {
   points: number
+  lowerBound: number
+  upperBound: number
   repeatedEvent: boolean
   riskBudget: number
   riskBudgetSetter: (newValue: number) => void
+  data: CalculatorData
 }): React.ReactElement {
   const points = props.points
 
-  const activeRiskLevel = howRisky(points, props.riskBudget)
+  const [displayCalculationExplanation, setCalculationExplanation] = useState(
+    false,
+  )
+
+  const locationRisk = calculateLocationPersonAverage(props.data) || 0
+  const personRiskEach = Math.round(
+    calculatePersonRiskEach(props.data, locationRisk) || 0,
+  )
+  const activityRisk = calculateActivityRisk(props.data)
+
+  const personRiskEachFormatted = personRiskEach.toLocaleString()
+  const pointsFormatted = displayPoints(points)
+  const pointsPercentFormatted = displayPercent(points)
+  const activityRiskFormatted = fixedPointPrecisionPercent(activityRisk)
+  const personCountSuffixFormatted =
+    props.data.personCount === 1 ? 'person' : 'people'
+  const frequencyFormatted = props.repeatedEvent
+    ? 'per week'
+    : 'each time you do it'
+  const lowerBoundFormatted = displayPoints(props.lowerBound)
+  const upperBoundFormatted = displayPoints(props.upperBound)
+  const budgetFormatted = displayPoints(props.riskBudget)
+  const budgetAnnualPercentFormatted = fixedPointPrecisionPercent(
+    props.riskBudget / 1000000,
+  )
+  const weekBudgetFormatted = getWeekBudget(props.riskBudget)
+  const budgetConsumptionFormatted = budgetConsumption(
+    props.points,
+    props.riskBudget,
+  )
+
+  const calculationBreakdown = (
+    <>
+      <span
+        className="expandable-header"
+        onClick={() =>
+          setCalculationExplanation(!displayCalculationExplanation)
+        }
+        aria-controls="calculation-explanation"
+        aria-expanded={displayCalculationExplanation}
+      >
+        {displayCalculationExplanation ? (
+          <>
+            <BsChevronDown /> How this was calculated
+          </>
+        ) : (
+          <>
+            <BsChevronRight /> Learn how this was calculated
+          </>
+        )}
+      </span>
+      <Collapse in={displayCalculationExplanation}>
+        <div id="calculation-explanation">
+          <div>
+            <h4>Calculation:</h4>
+            <div id="calculation-breakdown">
+              <code>
+                ({personRiskEachFormatted} Person Risk) x (
+                {activityRiskFormatted} Activity Risk) x (
+                {props.data.personCount} {personCountSuffixFormatted})<br />
+                <strong style={{ fontSize: '1.5em' }}>
+                  = ~{pointsFormatted} microCOVIDs {frequencyFormatted}
+                </strong>{' '}
+                <br />
+                (margin of error: {lowerBoundFormatted} to {upperBoundFormatted}
```

Comment:
I don't like "margin of error" because it is a technical term that is expressed as a percentage, rather than bounds. What happened to "probably between"? Or "confidence interval"?

Ok to leave for now.